frontstage/controllers/collection_exercise_controller.py

```diff
@@ -0,0 +1,42 @@
+import logging
+
+from flask import current_app as app
+import requests
+from structlog import wrap_logger
+
+from frontstage.exceptions.exceptions import ApiError
+
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+def get_collection_exercise(collection_exercise_id):
+    logger.debug('Attempting to retrieve collection exercise', collection_exercise_id=collection_exercise_id)
+    url = f"{app.config['COLLECTION_EXERCISE_URL']}/collectionexercises/{collection_exercise_id}"
+
+    response = requests.get(url, auth=app.config['COLLECTION_EXERCISE_AUTH'])
+
+    try:
+        response.raise_for_status()
+    except requests.exceptions.HTTPError:
+        logger.exception('Failed to retrieve collection exercise', collection_exercise_id=collection_exercise_id)
+        raise ApiError(response)
+
+    logger.debug('Successfully retrieved collection exercise', collection_exercise_id=collection_exercise_id)
+    return response.json()
+
+
+def get_collection_exercise_event(collection_exercise_id, tag):
+    logger.debug('Retrieving collection exercise event', collection_exercise_id=collection_exercise_id, tag=tag)
+    url = f"{app.config['COLLECTION_EXERCISE_URL']}/collectionexercises/{collection_exercise_id}/events/{tag}"
+
+    response = requests.get(url, auth=app.config['COLLECTION_EXERCISE_AUTH'])
+
+    try:
+        response.raise_for_status()
+    except requests.exceptions.HTTPError:
+        logger.exception('Failed to retrieve collection exercise event', collection_exercise_id=collection_exercise_id, tag=tag)
```

Comment:
Same thing about logger.exception logging at error level when we probably don't want to do that.